### PR TITLE
#160575946 implement feedback given in PR #160534388

### PR DESCRIPTION
**Pull request by nwashangai**
#### What does this PR do?
It refactors code
#### Description of Task to be completed?
* remove base64encode image file
* rename datatime to dateTime
* update eslint rule extension
* change ID generator extension
* change setOrder method name to makeOrder
* destruct makeOrder request body
#### How should this be manually tested?
* Not applicable
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
#160575946
#### Screenshots (if appropriate)
* None
#### Questions:
* None

### CODE REVIEW

File: src/controllers/OrderController.js [hunk 1]
```diff
@@ -33,4 +37,4 @@ class OrderController {
   }
 }
 
-module.exports = new OrderController();
+export let orders = new OrderController();
```
Comment (dogoyaro) [inline src/controllers/OrderController.js]: This should be `const`. Please research when to use `const` and when to use `let`

Comment (dogoyaro) [reply src/controllers/OrderController.js]: This should be a default export.

File: src/utils/updateOrder.js [hunk 2]
```diff
@@ -12,3 +12,5 @@ exports.updateOrder = (id, status) => {
   }
   return { count, updated };
 };
+
+export { updateOrder };
```
Comment (dogoyaro) [inline src/utils/updateOrder.js]: Let's sync on this. I prefer that we stick to the format for exporting prescribed in the es6 standard.

**Review by dogoyaro — CHANGES_REQUESTED** [review]

File: src/controllers/OrderController.js (hunk 1 above)
Comment (nwashangai) [reply src/controllers/OrderController.js]: Fix has been applied to this commit

File: src/app.js [hunk 3]
```diff
@@ -21,4 +21,4 @@ app.use((req, res, next) => {
 
 app.listen(process.env.PORT || 3000, () => {});
 
-module.exports = app; // for testing
+export { app };
```
Comment (dogoyaro) [inline src/app.js]: I think this is supposed to be `export default app`. Please read through the documentation to understand the difference

File: src/config/index.js [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
     PORT: process.env.PORT,
     SECRET: process.env.SECRET_KEY,
     POOL: {
     },
   },
-}
\ No newline at end of file
+};
+
+export { config };
```
Comment (dogoyaro) [inline src/config/index.js]: same as above

File: src/controllers/OrderController.js (hunk 1 above)
Comment (dogoyaro) [reply src/controllers/OrderController.js]: I can't see the fix @nwashangai

File: src/utils/updateOrder.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import { order } from '../models/store';
 
-exports.updateOrder = (id, status) => {
+export const updateOrder = (id, status) => {
```
Comment (dogoyaro) [inline src/utils/updateOrder.js]: This should be a default export.

**Comment by dogoyaro** [comment]
This is good to merge.

**Pull request merged**